sentinel/pkg/checker/dal/app.go

```diff
@@ -0,0 +1,89 @@
+package dal
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"bisonai.com/orakl/sentinel/pkg/alert"
+	"bisonai.com/orakl/sentinel/pkg/request"
+	"bisonai.com/orakl/sentinel/pkg/secrets"
+	"github.com/rs/zerolog/log"
+)
+
+const DefaultDalCheckInterval = 10 * time.Second
+
+type OutgoingSubmissionData struct {
+	Symbol        string   `json:"symbol"`
+	Value         string   `json:"value"`
+	AggregateTime string   `json:"aggregateTime"`
+	Proof         []byte   `json:"proof"`
+	FeedHash      [32]byte `json:"feedHash"`
+	Decimals      string   `json:"decimals"`
+}
+
+func Start() error {
+	interval, err := time.ParseDuration(os.Getenv("DAL_CHECK_INTERVAL"))
+	if err != nil {
+		interval = DefaultDalCheckInterval
+	}
+
+	chain := os.Getenv("CHAIN")
+	if chain == "" {
+		return errors.New("CHAIN not found")
+	}
+
+	key := secrets.GetSecret("DAL_API_KEY")
+	if key == "" {
+		return errors.New("DAL_API_KEY not found")
+	}
+
+	endpoint := fmt.Sprintf("https://dal.%s.orakl.network/api/v1", chain)
```

Comment:
this will need to be updated to 
```go
endpoint := fmt.Sprintf("https://dal.%s.orakl.network", chain)
```
since there is a PR that shortens the endpoint

Reply:
will fix after the pr is merged